aten/src/ATen/native/native_functions.yaml

```diff
@@ -1539,7 +1539,7 @@
 - func: kl_div(Tensor self, Tensor target, int reduction=Mean, bool log_target=False) -> Tensor
   use_c10_dispatcher: full
 
-- func: kl_div_backward(Tensor grad_output, Tensor self, Tensor target, int reduction=Mean) -> Tensor
+- func: kl_div_backward(Tensor grad_output, Tensor self, Tensor target, int reduction=Mean, bool log_target=False) -> Tensor
```

Comment:
Let's make this, and the other additions of `log_target`, kwarg only. (Add a `*` before it)